File: 110algoritmos/exercicios011/ex011-TypeScript/ex011.ts
let largura = Number(prompt("Digite a largura da parede:"));
let altura = Number(prompt("Digite a altura da parede:"));
const area = altura * largura;
const tinta = area/2
console.log(`Sua parede tem a dimensao de ${largura}x${altura} e sua area eh de ${area}m.`);
console.log(`Para pintar essa parede, vc precisara de ${tinta} de tinta`);

File: 110algoritmos/exercicios011/ex011-Rust/ex011.rs
use std::io;

fn main() {

    let mut largura:String = String::from("");
    let mut altura:String = String::from("");

    println!("Digite a largura da parede: ");
    io::stdin()
        .read_line(&mut largura)
        .expect("Erro ao ler a largura");
    let largura:f32 = largura.trim().parse().expect("Erro conversao largura");

    println!("\nDigite a altura da parede: ");
    io::stdin()
        .read_line(&mut altura)
        .expect("Erro leitura de altura");
    let altura:f32 = altura.trim().parse().expect("Erro conversao altura");

    let area = altura * largura;
    let tinta = area/2.0;

    println!("Sua parede tem a dimensao de {}x{}. E sua área e de {}m.", altura, largura, area);
    println!("Para pintar a parede precisará de {}l de tinta", tinta);


}